lib/Doctrine/ORM/Exception/PessimisticLockFailed.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Exception;
+
+final class PessimisticLockFailed extends \RuntimeException implements ORMException
+{
+    /**
+     * @return PessimisticLockFailed
+     */
+    public static function lockFailed()
```

Comment:
Return type maybe?

Reply:
Probably not in this PR, not directly related. :)